package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-toggle",
-  "version": "2.2.1",
+  "version": "2.3.0",
```

Comment:
I think you're right that this is a major upgrade. Since the library isn't backwards compatible, it should be 3.0. http://semver.org/#spec-item-8

I'll update shortly.